packages/react/src/Grid/Grid.tsx

```diff
@@ -22,17 +22,76 @@ export type GridColumnNumbers = {
   wide: GridColumnNumber
 }
 
-interface GridComponent
-  extends ForwardRefExoticComponent<PropsWithChildren<HTMLAttributes<HTMLDivElement>> & RefAttributes<HTMLDivElement>> {
+type GridDensity = 'low' | 'high'
```

Comment:
I'm not convinced the "internal app theme" should be a property on all components used. It should be a theme (provider) or different set of tokens. Perhaps we can investigate further?